Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/dialogs/GameSettingsDialog.java

```diff
@@ -64,14 +68,24 @@ public Dialog onCreateDialog(Bundle savedInstanceState)
                     SettingsActivity.launch(getActivity(), MenuTag.WIIMOTE, gameId);
                   break;
                 case 4:
-                  clearGameSettings(gameId);
+                  // Set as default ISO for GC, Wii clear game settings for else
```

Comment:
This seems very error prone to me... Is there no other way to tell what option was selected?  Not a big fan of those unnamed indices which are essentially magic numbers that can even change depending on the game file platform (!)

Reply:
Yeah, I think I found a better method.

Reply:
I moved my changes to earlier in the array to reduce ugliness. The cases aren't named because they change depending upon platform. I think the goal was to conditionally include Wiimote settings while keeping Clear Game Settings at the end of the array. I could separate the Game Properties `.setItems` by platform but then I'd need to duplicate the `switch`. There's only one case that differs between platforms so I'm thinking I should keep this as-is. Let me know if you disagree.